src/Designer/frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeListsPage/CodeListsPage.tsx

```
@@ -1,11 +1,56 @@
+  createCodeListMap,
+  deleteCodeListFromMap,
+  updateCodeListDataInMap,
+} from './utils';
+import { CodeListDataEditor } from './CodeListDataEditor';
+import type { CodeListMap } from './types/CodeListMap';
+import type { CodeListData } from './types/CodeListData';
 
 export type CodeListsPageProps = {};
 
-export function CodeListsPage(): ReactElement {
+export function CodeListsPage({}: CodeListsPageProps): ReactElement {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)